pkg/storage/edv/formatprovider/formatprovider.go

```diff
@@ -38,10 +43,24 @@ var (
 
 type marshalFunc func(interface{}) ([]byte, error)
 
+type MACComputer interface {
```

Comment:
I realize this name is a little bit funny. Let me know if you have any suggestions.

Reply:
`MACDigester` is better as we are usually talking about a message digest when referring to Mac.

Reply:
Done!